src/script/zcash_script.h

```diff
@@ -99,6 +99,28 @@ EXPORT_SYMBOL int zcash_script_verify(
     uint32_t consensusBranchId,
     zcash_script_error* err);
 
+/// Returns the number of transparent signature operations in the
+/// transparent inputs and outputs of the precomputed transaction
+/// pointed to by preTx.
+///
+/// Returns UINT_MAX on error.
```

Comment:
This is a little unusual as an error sentinel, but it's fine because the 2MB bound on transaction size prevents a single transaction from ever containing this many sigops.

Reply:
Thanks, I'll explain why in the docs.

```suggestion
/// Returns UINT_MAX on error, so that invalid transactions don't pass the Zcash consensus rules.
```